README.md

```diff
@@ -1182,31 +1191,31 @@ The API key must have the following scopes:
 #### Usage
 
 ```sh
-yarn tr-manual-enrichment-push-identifiers --auth=$TRANSCEND_API_KEY
+yarn tr-manual-enrichment-pull-identifiers --auth=$TRANSCEND_API_KEY
```

Comment:
Does this mean we used to have a push endpoint which has been replaced with a pull endpoint?

Reply:
no i just copy pasta'd here